src/MacAppBundle/CMakeLists.txt

```diff
@@ -1,3 +1,27 @@
+
+#
+# Build and install QuickLook for FCStd files (org.freecad.fcstd)
+# This is used by Homebrew and Conda/Mamabe scripts alike,
+#
+if(FREECAD_CREATE_MAC_APP OR (APPLE AND BUID_WITH_CONDA))
```

Comment:
`BUID` or `BUILD` ?

Reply:
Also, line 4 looks like it should be `Mamba` not `Mamabe`

Reply:
Sigh. BUILD_WITH_CONDA it is. Sliped testing thanks to not clearing artefacts correctly.

Reply:
Fixed.

Reply:
See https://github.com/FreeCAD/FreeCAD/pull/7491#discussion_r1002753490

Reply:
Typo also fixed.

Reply:
#7491 It's confined to a single issue that needs to be implemented in two redundant build systems. If you prefer, I'll be happy to separate this into three PRs. It's in the history anyway as I implemented Homebrew first and then just added the Conda/Mamba stuff, which is just line 6.